appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/base/LocalStore.kt

```diff
@@ -0,0 +1,37 @@
+package org.kinecosystem.appsdiscovery.base
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class LocalStore(context: Context, name:String){
+
+    private val sharedPreferences:SharedPreferences = context.applicationContext.getSharedPreferences(name, Context.MODE_PRIVATE)
+
+    fun updateString(key:String, value:String){
+        sharedPreferences.edit().apply{
+            putString(key, value)
+            apply()
+        }
```

Comment:
if you use android ktx you can minimize the code to:
```
sharedPreferences.edit {
    putString("key", value)
}
```

See [https://developer.android.com/kotlin/ktx#kotlin](https://developer.android.com/kotlin/ktx#kotlin)

Reply:
we are thinking or removing the ktx lib